corehq/blobs/management/commands/import_blob_zip.py

```diff
@@ -24,7 +24,7 @@ def handle(self, zipname, **options):
         for filename in from_zip.namelist():
             bucket = '/'.join(filename.split('/')[:-1])
             identifier = filename.split('/')[-1]
-            blob = cStringIO.StringIO(from_zip.read(filename))
+            blob = io.StringIO(from_zip.read(filename))
```

Comment:
This should use `BytesIO`. [`ZipFile.read()` returns bytes](https://docs.python.org/2/library/zipfile.html#zipfile.ZipFile.read).